quickstart/101-application-gateway/main.tf

```diff
@@ -0,0 +1,176 @@
+terraform {
```

Comment:
Please move the terraform and provider blocks to a providers.tf file.

Reply:
@vhorne please rename this file to providers.tf ( with an "s" on the end) then it will be good to merge.